unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Utils/CoordinateUtils.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.EventSystems;
+using UnityEngine.Serialization;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+public class CoordinateUtils
+{
+    public const string COORDINATE_MATCH_REGEX = "[-]?\\d{1,3},[-]?\\d{1,3}";
+    public const int MAX_X_COORDINATE = 163;
+    public const int MIN_X_COORDINATE = -150;
+    public const int MAX_Y_COORDINATE = 158;
+    public const int MIN_Y_COORDINATE = -150;
+
+    public static bool IsCoordinateInRange(int x, int y) { return (x <= MAX_X_COORDINATE && x >= MIN_X_COORDINATE && y <= MAX_Y_COORDINATE && y >= MIN_Y_COORDINATE); }
+
+    public static List<string> GetTextCoordinates(string text)
+    {
+        Regex filter = new Regex(COORDINATE_MATCH_REGEX);
+        List<string> matchingWords = new List<string>();
+        foreach (var item in text.Split(' '))
+        {
+            var match = filter.Match(item.ToString());
+            int x, y;
```

Comment:
Although it's not specified anywhere, our implicit convention is having the declarations in lines rather than using multiple declaration.